src/NLog/Config/LoggingConfiguration.cs

```diff
@@ -249,12 +249,13 @@ public void Install(InstallationContext installationContext)
                 }
                 catch (Exception exception)
                 {
+                    InternalLogger.Error(exception, "Install of '{0}' failed.", installable);
                     if (exception.MustBeRethrown())
                     {
                         throw;
                     }
 
-                    installationContext.Error("'{0}' installation failed: {1}.", installable, exception);
+                    installationContext.Error("Install of '{0}' failed: {1}.", installable, exception);
```

Comment:
This may be a breaking change. Remember, the old `MustBeRethrown()` is more like the current `MustBeRethrownImmediately()`.  
If `LogManager.ThrowExceptions` is `true`, then the installation context logging would not occur, as it would have before.